nidirect_custom_blocks/src/Plugin/Block/NIDirectArticleTeasersByTopic.php

```diff
@@ -94,6 +105,30 @@ public function build() {
     return $build;
   }
 
+  /**
+   * Utility function to sort article teasers.
+   */
+  private function userSortResults($node, array $results) {
```

Comment:
I'd recommend type hinting the `$node`. Something like:

```diff
- private function userSortResults($node, array $results) {
+ private function userSortResults(NodeInterface $node, array $results) {
```
With the obligatory function annotations too.

Reply:
that's done